addons/l10n_ro_efactura/models/res_company.py

```diff
@@ -33,17 +34,27 @@ def _compute_l10n_ro_edi_callback_url(self):
             else:
                 company.l10n_ro_edi_callback_url = False
 
+    def _l10n_ro_edi_log_message(self, cr, message: str, func: str):
+        self.env['ir.logging'].sudo().create({
+            'name': 'l10n_ro_edi_log',
+            'type': 'server',
+            'level': 'INFO',
+            'dbname': self.env.cr.dbname,
+            'message': message,
+            'func': func,
+            'path': '',
+            'line': '1',
+        })
+        cr.commit()
```

Comment:
We can't pass the cursor. It's a security risk.

Also we are doing exactly the same thing as before. `cr` is still `self.env.cr`. It should look more something like this :

```suggestion
    def _l10n_ro_edi_log_message(self, cr, message: str, func: str):
        with self.pool.cursor() as cr:
            self.with_env(self.env(cr=cr)).env['ir.logging'].sudo().create({
                ...
            })
```